### PR TITLE
Upgrade to Angular 18, and drop support for 17

### PR DESCRIPTION
**Pull request by jpzwarte**

### CODE REVIEW

File: packages/angular/.storybook/main.ts [hunk 1]
```diff
@@ -1,20 +1,9 @@
-import type { StorybookConfig } from '@storybook/angular';
+import { type StorybookConfig } from '@storybook/angular';
 
 const config: StorybookConfig = {
```
Comment (anna-lach) [inline packages/angular/.storybook/main.ts:3]: The Angular components/wrappers are not being loaded in the storybook?

image

File: .changeset/perfect-hotels-flow.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@sl-design-system/angular': major
+---
+
+Upgrade to Angular 18 and drop support for 17
```
Comment (Diaan) [inline .changeset/perfect-hotels-flow.md:5]: is this checked with the cfa's?

**Review by Diaan — APPROVED** [review]

File: packages/angular/.storybook/main.ts (hunk 1 above)
Comment (jpzwarte) [reply packages/angular/.storybook/main.ts:3]: Hmm, apparently this happens on `main` as well. It worked fine when i ran Angular storybook locally. I'll investigate.

File: .changeset/perfect-hotels-flow.md (hunk 2 above)
Comment (jpzwarte) [reply .changeset/perfect-hotels-flow.md:5]: No, but it's best practice for Angular libraries to support n + n-1. So now that Angular 19 is out, 18 + 19, but not 17.

File: packages/angular/.storybook/main.ts (hunk 1 above)
Comment (jpzwarte) [reply packages/angular/.storybook/main.ts:3]: Fixed it! Apparently you to explicit define a `constructor` and call `super()` when you extend the `CePassthrough` directive.

**Pull request merged**